### PR TITLE
Makefile update, test update, documentation detail

### PR DESCRIPTION
**Pull request by pouicr**
Here is some improvement.
Maybe too many at the same time, sorry...

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,11 @@ all: build
 
 volume:
 	echo "Building $(APPNAME) volumes..."
-	sudo docker run -v $(PWD):/opt/$(APPNAME) -v ~/var/$(APPNAME):/var/opt/$(APPNAME) --name $(APPNAME)_volumes busybox true
+	sudo docker run -v $(PWD)/scripts:/var/opt/webhookd/scripts --name $(APPNAME)_volumes busybox true
```
Comment (ncarlier) [inline Makefile]: Could you keep the first mount point. It's used to run the container in dev mode (aka edit file onto the host)

File: Makefile [hunk 2]
```diff
@@ -14,7 +14,7 @@ define docker_run_flags
 --rm \
 -v /var/run/docker.sock:/var/run/docker.sock \
 --env-file $(PWD)/etc/env.conf \
--P \
+-p 8081:8080 \
```
Comment (ncarlier) [inline Makefile]: Can you let the random port directive? I usually run this container along side a service discovery container.

File: Makefile [hunk 3]
```diff
@@ -22,7 +22,11 @@ all: build
 
 volume:
 	echo "Building $(APPNAME) volumes..."
-	sudo docker run -v $(PWD):/opt/$(APPNAME) -v ~/var/$(APPNAME):/var/opt/$(APPNAME) --name $(APPNAME)_volumes busybox true
+	sudo docker run -v $(PWD)/scripts:/var/opt/webhookd/scripts --name $(APPNAME)_volumes busybox true
+
+key:
+	$(eval docker_run_flags += -v $(PWD)/ssh:/root/.ssh)
+	echo "Add techKey"
```
Comment (ncarlier) [inline Makefile]: Good point! Please rename "techkey": "Add private deploy key."

File: scripts/gitlab/build.sh [hunk 4]
```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
```
Comment (ncarlier) [inline scripts/gitlab/build.sh]: You should not commit this script. It's a bit too specific need.

**Comment by pouicr** [comment]
Here it is

**Comment by ncarlier** [comment]
Great. Thanks!

**Pull request merged**